package.js

```diff
@@ -0,0 +1,20 @@
+Package.describe({
+  name: 'useful:collection-mocks',
+  version: '0.0.1',
+  summary: 'Adds a method to Mongo.Collection\'s prototype that mocks the results of a db operation',
+  git: 'https://github.com/usefulio/meteor-collection-mocks.git',
```

Comment:
I've renamed the repo to drop the `meteor-` prefix (at @ianserlin's suggestion), let's update this url to point to the renamed repo.